src/scales/scale.time.js

```diff
@@ -175,8 +175,20 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var majorUnitStart = startTick;
+			if (options.majorUnit) {
+				majorUnitStart = moment(startTick).add(1, options.majorUnit).startOf(options.majorUnit);
```

Comment:
if startTick is already at a majorUnit then I think you'll end up at the next one. do you really want something like:

    majorUnitStart = moment(startTick).startOf(options.majorUnit) == moment(startTick) ? moment(startTick) : moment(startTick).add(1, options.majorUnit).startOf(options.majorUnit);